front/src/reducers/editor/index.ts

```diff
@@ -274,11 +273,10 @@ export function save(
   return saveOperations(infraID, payload);
 }
 
-export type EditorActions =
-  | editorSliceActionsType['loadDataModelAction']
-  | editorSliceActionsType['selectLayers']
-  | editorSliceActionsType['updateFiltersIssueAction']
-  | editorSliceActionsType['updateTotalsIssueAction']
-  | ActionSave;
+export const editorActions = editorSlice.actions;
+
+export type EditorActions = typeof editorActions | ActionSave;
```

Comment:
To avoid import mistakes with the previous one
```suggestion
export type EditorActionsType = typeof editorActions | ActionSave;
```